prime-router/metadata/schemas/iPatientCare/iPatientCare-covid-19.schema

```diff
@@ -55,7 +57,7 @@ elements:
       csvFields: [{name: Patient phone}]
 
     - name: patient_dob
-      csvFields: [{name: Birth Date, format: "M/d/yyyy H:nn"}]
+      csvFields: [{name: Birth Date, format: "M/d/yyyy"}]
```

Comment:
I am guessing changing this format can be a problem for the sender.  @jimduff-usds?

Reply:
Since we do not own the schema, could we leave it as it was?

Reply:
Ugh.  This will require a bit of research here.  Follow up with someone about this format, but the Element formatter does not support this as it is a date and not a date time.  So, why are we using this schema for fake data generation?

Reply:
I asked @jbiskie regarding the schema.  And, he told me as below:
---
"We need that format to align with how the iPatientCare report sends the DOB.  Is that a new issue?  We've been able to accept it so far...
On the second one, are these new integration tests that I can run to see what's happening?  If so, are they in a branch or master?" 
---
I will ask @jimduff-usds about how we can go about to resolve the bug.

Reply:
During the Wed PR review meeting, we went through the Element.kt code.  toNormalized and checkError both take care of this weird use case, but toFormatted does not.  This is an existing bug that can be fixed by making that code consistent with the other functions.

Reply:
I fixed the code in Element.kt.